### PR TITLE
Update documentation for the new CRD helm chart

### PR DESCRIPTION
**Pull request by raulcabello**
fixes kubewarden/docs#53

### CODE REVIEW

File: src/quick-start.md [hunk 1]
```diff
@@ -18,12 +18,16 @@ Currently, the chart depends on cert-manager. Make sure you have [`cert-manager`
 ```console
 helm repo add kubewarden https://charts.kubewarden.io
 kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml
-helm install --wait --namespace kubewarden --create-namespace kubewarden-controller kubewarden/kubewarden-controller
+helm install --create-namespace -n kubewarden kubewarden-crds kubewarden/kubewarden-crds
```
Comment (flavio) [inline src/quick-start.md:21]: I don't think you have to create a namespace here, this is just installing the CRDs. Obviously it won't hurt, but maybe we should move it to the next line?

**Review by flavio — APPROVED** [review]
LGTM

File: src/quick-start.md (hunk 1 above)
Comment (raulcabello) [reply src/quick-start.md:21]: it will be installed in the default namespace if we don't provide one. So I think we should use the kubewarden ns

Comment (ereslibre) [reply src/quick-start.md:21]: CRD's are cluster wide though, so it should not matter

**Review by ereslibre — APPROVED** [review]

**Review by jvanz — APPROVED** [review]

File: src/quick-start.md (hunk 1 above)
Comment (flavio) [reply src/quick-start.md:21]: Good point

**Pull request merged**